browser/scripts/application.js

```diff
@@ -1304,7 +1285,7 @@ Application.prototype.onKeyDown = function(e) {
 		}
 		if(e.keyCode === 66) // CTRL+b
 		{
-			this.toggleLeftPane();
+			this.onBtnPresetsClicked();
```

Comment:
this should actually toggle everything in the UI and VP layer and show only the output, as it does now - it also hides the chat pane (in develop)